manifests/pagerduty.pp

```diff
@@ -21,7 +21,8 @@
     type    => 'pipe',
     source  => 'puppet:///modules/sensu_handlers/pagerduty.rb',
     config  => {
-      teams => $teams,
+      teams       => $teams,
+      environment => $environment,
```

Comment:
Or I guess we can ignore this PR. I'll update check's region attr with the way you use it, env+region. thanks for your time. If you are already using it and have no issues, then I guess there is no reason to worry.